src/style.css

```diff
@@ -0,0 +1,139 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  --white-color: #fff;
+  --border-color: #f3f3f3;
+  --title-color: #060b35;
+  --subtitle-color: #616070;
+  --color-for-link: #00acdc;
+}
+
+body,
+h2,
+h3,
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  padding: 32px 16px 16px;
+  width: 166px;
```

Comment:
just add `box-sizing: border-box;` to `*` selector and use `width: 200px` here

Reply:
is not fixed

Reply:
Thanks for advise, but now I don't have style.css file, so I made changes at the card.css file.